### PR TITLE
liballoc, libunicode: Inline police.

### PR DESCRIPTION
**Pull request by pcwalton**
r? @cmr

### CODE REVIEW

File: src/liballoc/arc.rs [hunk 1]
```diff
@@ -210,6 +210,18 @@ impl<T> Arc<T> {
         // contents.
         unsafe { &**self._ptr }
     }
+
+    // Non-inlined part of `drop`.
+    unsafe fn drop_slow(&mut self) {
```
Comment (huonw) [inline src/liballoc/arc.rs]: Should this be `#[inline(never)]`?

File: src/libunicode/char.rs [hunk 2]
```diff
@@ -41,6 +41,426 @@ pub use normalize::{decompose_canonical, decompose_compatible, compose};
 pub use tables::normalization::canonical_combining_class;
 pub use tables::UNICODE_VERSION;
 
+#[cfg(stage0)]
```
Comment (Manishearth) [inline src/libunicode/char.rs]: Should this have a `// SNAP` comment?

**Pull request merged**